src/Http/BaseApplication.php

```diff
@@ -82,7 +82,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
         $cakeResponse = $this->getDispatcher()->dispatch($cakeRequest, $cakeResponse);
 
         // Convert the response back into a PSR7 object.
-        return $next($request, ResponseTransformer::toPsr($cakeResponse));
+        return ResponseTransformer::toPsr($cakeResponse);
```

Comment:
One of your examples in your talk is that you could test the app in isolation. Another reason could be composing 2 different apps.